app/backend/src/couchers/servicers/auth.py

```diff
@@ -204,7 +204,7 @@ def CompleteSignup(self, request, context):
 
             birthdate = parse_date(request.birthdate)
 
-            if not birthdate or birthdate >= today():
+            if not birthdate or birthdate >= required_birthdate():
                 context.abort(grpc.StatusCode.INVALID_ARGUMENT, errors.INVALID_BIRTHDATE)
```

Comment:
please find the `INVALID_BIRTHDATE` string in errors and change it to something like `You must be at least 18 years old to sign up.`